src/components/Button/Button.tsx

```diff
@@ -0,0 +1,24 @@
+import cn from 'classnames';
+
+import styles from './Button.module.css';
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  priority: string;
+  children: React.ReactNode;
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+export function Button({priority, children, onClick, ...attr}: ButtonProps) {
+  return (
+    <button
+      {...attr}
+      className={cn({
+        [styles[`${'button_button__' + priority}`]]: priority,
```

Comment:
template literals can sometimes be unnecessary

```suggestion
        [styles['button_button__' + priority]]: priority,
```